recipe/meta.yaml

```
@@ -1,37 +1,34 @@
-{% set version = "2.12.1" %}
-{% set estimator_version = "2.12.0" %}
+{% set version = "2.13.0" %}
 
 package:
   name: tensorflow-split
   version: {{ version }}
 
 source:
   - url: https://github.com/tensorflow/tensorflow/archive/refs/tags/v{{ version.replace(".rc", "-rc") }}.tar.gz
-    sha256: 6bc4600cc0b88e9e40f1800096f5bddbbd3b6e5527a030dea631b87f2ae46b5b
+    sha256: e58c939079588623e6fa1d054aec2f90f95018266e0a970fd353a5244f5173dc
     patches:
       - patches/0001-loosen-requirements.patch
       - patches/0002-fix-genproto.patch
       - patches/0003-fix-absl-py.patch
       - patches/0004-Add-additional-absl_synchronization-linkage-to-gRPC.patch
-      - patches/0005-add-missing-macro-for-CUDA_VERSION.patch
-      # backport https://github.com/tensorflow/tensorflow/commit/2eae48506773587a7e74dd7064aa0fa9f0c37fbf
-      # to not have to carry an upper bound for wrapt-version
-      - patches/0006-Add-TypeError-catch-wrapt-1.15.0rc-throws-TypeError-.patch
-      # backport https://github.com/tensorflow/tensorflow/commit/ad938db0da16d00d7f14da3de2a4abb97b1bc340
-      # for compatibility with newer abseil
-      - patches/0007-Update-Abseil-to-the-latest-LTS-replace-or-with-in-d.patch
-      - patches/0008-Add-missing-any_invocable-target-to-absel-bazel.patch
-      - patches/0009-Fix-missing-abseil-linkages.patch
+      - patches/0005-Add-missing-any_invocable-target-to-absel-bazel.patch
+      - patches/0006-Fix-missing-abseil-linkages.patch
+      # backport https://github.com/tensorflow/tensorflow/pull/61134
+      - patches/0007-unpin-gast-dependency-for-pip_package.patch
+      - patches/0008-Use-basename-instead-of-realpath.patch
+      - patches/0009-Fix-protobuf_python-for-systemlibs.patch
+      - patches/0010-Add-absl_log-systemlib.patch
+      - patches/0001-Omit-linking-to-layout_proto_cc-if-protobuf-linkage-.patch
 
-  - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ estimator_version.replace(".rc", "-rc") }}.tar.gz
-    sha256: 86c75e830c6ba762d0e3cf04c160096930fb12a992e69b3f24674b9f58902063
+  - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ version.replace(".rc", "-rc") }}.tar.gz
+    sha256: 4175e9276a1eb8b5e4e876d228e4605871832e7bd8517965d6a47f1481af2c3e
     folder: tensorflow-estimator
 
 build:
   number: 0
   skip: true  # [win]
   skip: true  # [python_impl == 'pypy']
-  skip: true  # [cuda_compiler_version in ("10.2", "11.0", "11.1")]
 
 requirements:
   build:
```

```suggestion
  build:
    - perl                                   # [linux]
```
I needed perl during the build process